CETCompression.md

```diff
@@ -368,6 +370,43 @@ def groupByIgnoringDigits(start: Long, end: Long, base: Int, numDigits: Int): Ve
 }
 ```
 
+## Adaptor Point Computation Optimizations
+
+Computing a large number of adaptor points for events with outcome composed of many digits is a computationally intensive process.
```

Comment:
probably worth noting for context that in reference implementations, signature point computation accounts for a significant proportion of runtime

Reply:
updated but not entirely satisfied with the wording so let me know if you have better suggestions

Reply:
good enough for me :)